tests/util/test_init.py

```diff
@@ -41,6 +41,10 @@ def test_slugify():
     assert util.slugify("Tèst_äöüß_ÄÖÜ") == "test_aouss_aou"
     assert util.slugify("影師嗎") == "ying_shi_ma"
     assert util.slugify("けいふぉんと") == "keihuonto"
+    assert util.slugify("$") == "unknown"
+    assert util.slugify("$$$") == "unknown"
+    assert util.slugify("$something") == "something"
+    assert util.slugify("") == ""
```

Comment:
Why not add the test cases you described in the PR?

Reply:
I thought `$` would be more relatable, but good idea, I've added Ⓐ and ⓑ as well.